ovn_cluster.sh

```diff
@@ -707,12 +707,15 @@ create_fake_vm() {
     ipv6_addr=\$7
     ipv6_gw=\$8
     ip netns add \$name
-    ovs-vsctl \
-      -- add-port br-int \$name \
-      -- set interface \$name type=internal \
-      -- set Interface \$name external_ids:iface-id=\$iface_id
+    ip link add \$name-p type veth peer name \$name
     ip link set \$name netns \$name
     ip netns exec \$name ip link set lo up
+    ip link set \$name-p up
+
+    ovs-vsctl \
+      -- add-port br-int \$name-p \
+      -- set Interface \$name-p external_ids:iface-id=\$iface_id
+    ip netns exec \$name ip link set lo up
```

Comment:
`lo` is already up in namespace `$name`, it was set above.  I don't think we need to do it again.